core/state_transition.go

```diff
@@ -220,80 +228,35 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 	}
 
 	var (
-		evm = st.evm
 		// vm errors do not effect consensus and are therefore
 		// not assigned to err, except for insufficient balance
 		// error.
 		vmerr error
 	)
 
-	to := "<nil>"
-	if msg.To() != nil {
-		to = msg.To().Hex()
-	}
-
-	executionMgrTime := st.evm.Time
-	if executionMgrTime.Cmp(big.NewInt(0)) == 0 {
-		executionMgrTime = big.NewInt(1)
-	}
-
-	// TODO: queue origin is always 0 with current version of em
-	queueOrigin := big.NewInt(0)
-
-	l1MessageSender := msg.L1MessageSender()
-	if l1MessageSender == nil {
-		addr := common.HexToAddress("")
-		l1MessageSender = &addr
-	}
-
-	log.Debug("Applying transaction", "from", sender.Address().Hex(), "to", to, "nonce", msg.Nonce(), "l1MessageSender", l1MessageSender.Hex(), "data", hexutil.Encode(msg.Data()))
-
 	if contractCreation {
-		// Here we are going to call the EM directly
```

Comment:
All of this is now handled in `toExecutionManagerRun`, so not needed here anymore.

Reply:
awesome refactor